ozaria/site/styles/common/common.scss

```diff
@@ -1,186 +1,188 @@
 @import "ozaria/site/styles/common/variables";
+@import "app/styles/utils";
 
+@if $is-ozaria {
```

Comment:
are we merging this file with any other one? Why the flag?

Reply:
@mrfinch we need the flag, because we although we require this file under isOzaria flag, it seems webpack loads its content regardless in coco.